### DISCUSSION TITLE
crouton 2.0

### DISCUSSION THREAD

**Discussion by DeviateBacon3** [dnschneid/crouton#4723] [Ideas]
I am makeing crouton 2.0 heres the link https://github.com/DeviateBacon3/crouton-2.0

**Comment by DennisLfromGA** [comment]
This is *NOT* crouton **2.0** since it downloads the *original* crouton installer and uses it.

There is *NO* difference between this version and the original.

Closing discussion.

-**DennisLfromGA**
